Functions.Templates/Templates/SqlInputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = async function (context, results) {
+    if (!!results && results.length > 0) {
```

Comment:
Why wouldn't we log if the results are 0? That's a perfectly valid response

(and can results ever be undefined? That seems like that shouldn't be a valid case. Are there other examples you're following that do this kind of check?)

Reply:
yeah am following the cosmosDB example.

Reply:
I don't think we should assume they've done everything correctly (I've seen a lot of weird stuff in their examples) and it doesn't make sense to me to return undefined. Our code will always return *something* - even if it's just an empty list. 

And there's other samples like the blob trigger that don't do this so I think it should be fine to remove and help keep things cleaner. 

@chlafreniere @dzsquared ?

Reply:
results should not be undefined (bad things happened), but results.length could absolutely be 0.  instead of relying on logging, let's add an http trigger to this template and return the results.  (the logging can stay)

Reply:
@MaddyDev You still have the results check - are you not going to remove it? This whole thing can just be simplified to 

```javascript
module.exports = async function (context, results) {
    context.log('Query results: ', JSON.stringify(results));
    return {
        status: 200,
        body: results
    };
}
```

Reply:
the check is used just for logging, am returning the results either way. It can be simplified to above, @dzsquared want me to be consistent and do the above?

Reply:
We want these samples to be useful and simple enough for users - having all these extra checks isn't necessary and just a distraction from the actual core part of the code that's doing useful stuff IMO